infraagent/agent/cmd/agent/rootcmd.go

```diff
@@ -213,5 +219,9 @@ func validateConfigs() error {
 			}
 		}
 	}
+	hostIfaceMTU := viper.GetInt("hostIfaceMTU")
+	if hostIfaceMTU < 46 || hostIfaceMTU > 1500 {
```

Comment:
hostIfaceMTU <576 

576 bytes is the minimum IPv4 packet (datagram) size that any node must be able to handle. (RFC 791)

Reply:
fixed in later commit